common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -70,21 +70,9 @@
 <%def name="require_module(module_name, class_name)">
     <script type="text/javascript">
         (function (require) {
-            % if settings.REQUIRE_DEBUG:
-                require(['${module_name}'], function (${class_name}) {
-                    ${caller.body()}
-                });
-            % else:
-                ## The "raw" parameter is specified to avoid the URL from being further maninpulated by
-                ## static_replace calls (as woudl happen if require_module is used within courseware).
-                ## Without specifying "raw", a call to static_replace would result in the MD5 hash being
-                ## being appended more than once, causing the import to fail in production environments.
-                require(['${staticfiles_storage.url(module_name + ".js") + "?raw"}'], function () {
-                    require(['${module_name}'], function (${class_name}) {
-                        ${caller.body()}
-                    });
-                });
-            % endif
+            require(['${module_name}'], function (${class_name}) {
```

Comment:
I don't remember how to reproduce the issue that I was working around, though it should be pretty obvious on a sandbox if it isn't working (I'm not sure if bok choy tests can be counted on to find the issue).